common/src/main/proto/TransportMessages.proto

```diff
@@ -126,7 +126,8 @@ message PbRegisterWorker {
   int32 fetchPort = 4;
   int32 replicatePort = 5;
   map<string, PbDiskInfo> disks = 6;
-  string requestId = 9;
+  string requestId = 7;
+  map<string, PbResourceConsumption> userResourceUsage = 8;
```

Comment:
Register worker means that the worker is just initialized and empty. So I think there is no need to report empty userResourceUsage unless worker recovering saved that information.

Reply:
> Register worker means that the worker is just initialized and empty. So I think there is no need to report empty userResourceUsage unless worker recovering saved that information.

But if it's recovered from graceful shutdown?

Reply:
OK, it's reasonable.